features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -899,12 +906,138 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
+                        ".description", { state: enableStatus ? "enabled" : "disabled" }),
+                    level: AlertLevels.SUCCESS,
+                    message: t("applications:notifications.disableApplication.success.message", {
+                        state : enableStatus ? "enabled" : "disabled" })
+                }));
+
+                setShowDisableConfirmationModal(false);
+                onUpdate(application.id);
+            })
+            .catch((error: AxiosError) => {
+                if (error.response && error.response.data && error.response.data.description) {
+                    dispatch(addAlert({
+                        description: error.response.data.description,
+                        level: AlertLevels.ERROR,
+                        message: t("applications:notifications.disableApplication.error" +
+                            ".message")
+                    }));
+
+                    return;
+                }
+
+                dispatch(addAlert({
+                    description: t("applications:notifications.disableApplication" +
+                        ".genericError.description", { state: enableStatus ? "enable" : "disable" }),
+                    level: AlertLevels.ERROR,
+                    message: t("applications:notifications.disableApplication.genericError" +
+                        ".message")
+                }));
+            })
+            .finally(() => {
+                setIsDisableInProgress(false);
+            });
+    };
+
+    const applicationsUpdateScopes: string[] = featureConfig?.applications?.scopes?.update;
+
     const MyAccountOverviewTabPane = (): ReactElement => (
-        <ResourceTab.Pane controlledSegmentation>
-            <MyAccountOverview/>
-        </ResourceTab.Pane>
+        <>
+            <ResourceTab.Pane controlledSegmentation>
+                <MyAccountOverview/>
+            </ResourceTab.Pane>
+            <Divider hidden />
+            <Show
+                when={ applicationsUpdateScopes }
+            >
+                <DangerZoneGroup
+                    sectionHeader={ t("applications:dangerZoneGroup.header") }
+                >
+                    <DangerZone
+                        actionTitle={ t("applications:dangerZoneGroup.disableApplication.actionTitle",
+                            { state: application.applicationEnabled ? t("common:disable") : t("common:enable") }) }
+                        header={ t("applications:dangerZoneGroup.disableApplication.header",
+                            { state: application.applicationEnabled ? t("common:disable") : t("common:enable") } ) }
+                        subheader={ application.applicationEnabled
+                            ? t("applications:dangerZoneGroup.disableApplication.subheader")
+                            : t("applications:dangerZoneGroup.disableApplication.subheader2") }
+
+                        onActionClick={ undefined }
```

Comment:
Is passing undefined here intentional? this will cause non functioning buttons, wouldn't it?

Reply:
It is toggle. So need need to act upon click right?